subcommand/server-acl-init/rules_test.go

```diff
@@ -518,25 +518,25 @@ operator = "write"`,
 			false,
 			true,
 			`
-  node_prefix "" {
-     policy = "write"
-  }
-  service_prefix "" {
-     policy = "write"
-  }`,
+node_prefix "" {
+  policy = "write"
+}
+service_prefix "" {
+  policy = "write"
+}`,
 		},
 		{
 			"Namespaces are enabled, health checks controller enabled",
 			true,
 			true,
 			`
 operator = "write"
+node_prefix "" {
```

Comment:
The indentation of these tests might need to have their indentations updated if you're ok with the above changes